modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -134,6 +134,28 @@
             <groupId>javax.validation</groupId>
             <artifactId>validation-api</artifactId>
         </dependency>
-{{/useBeanValidation}}        
+{{/useBeanValidation}}
+{{#virtualService}}
+<!-- START Virtual Service API support -->
+		<dependency>
+			<groupId>io.virtualan</groupId>
+			<artifactId>virtualan-plugin</artifactId>
+			<version>1.0.0</version>
+		</dependency>
+		
+		<!-- HyperSQL DB -->
+		<dependency>
+			<groupId>org.hsqldb</groupId>
+			<artifactId>hsqldb</artifactId>
+			<version>${hsqldb.version}</version>
+		</dependency>
+
+		<dependency>
+		    <groupId>org.springframework.boot</groupId>
+		    <artifactId>spring-boot-starter-data-jpa</artifactId>
+		    <version>1.5.9.RELEASE</version>
+		</dependency>
+    <!-- END Virtual Service API support -->
+{{/virtualService}}       
```

Comment:
To make it consistent with the rest of the pom file, please use spaces instead of tabs

Reply:
Removed the Tab spaces